TABSwiftLayout.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name         = 'TABSwiftLayout'
-  s.version      = '1.0'
-  s.platform	 = :ios, '8.0'
+  s.version      = '1.1'
+  s.platforms     = { :ios => "8.0", :osx => "10.10" }
```

Comment:
Untested -- so I didn't include it.

I have this lib used in OSX and iOS so I know it works. But haven't checked for compatibility with tvOS.